.gitignore

```diff
@@ -5,3 +5,4 @@ bin/
 .idea
 .DS_Store
 lock.json
+main
```

Comment:
Why do need to add main in this .gitignore? If this is not common for everyone, I suggest keeping it locally your local dev.

Reply:
sure.